### DISCUSSION TITLE
Available on backorder ?

### DISCUSSION THREAD

**Discussion by mrgoodcat** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#94] [General]
Hi, 
most parts of your items are "available in backorder". What this mean exactly ? Like i have to purchase them and then i'm sure that in XX weeks i will receive the items ? There is an extimation about XX supposing to buy today?
Thanks

**Comment by alexruffell** [comment]
My understanding means that you can order it, but it is not in stock. Backordered, instead, is something you cannot order while not in stock. This may have something to do with the time it takes for the parts to be available and how long the payment/authorization is valid for. I believe you can keep an authorization on a credit card live for 30 days after which you must ask the person placing the order to place it again in order to renew the authorization otherwise you cannot charge upon shipping after 30 days the order was placed. This would indicate to me that "Available in backorder" means the shipment should happen before 30 days pass, and "backordered" likely means the parts will be available beyond the 30 day mark. However, I can't speak for circuitsetup but in a previous job this was the issue we had to deal with.
